build/crd-samples/devices/devicemockmodel.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: devices.kubeedge.io/v1alpha1
```

Comment:
Please rename the file to indicate that this is a mock bluetooth device.
Applicable to device instance as well

Reply:
The location of file is now changed to tests/e2e/mapper/bluetooth/crds